lib/ui/compositing/scene_builder.cc

```diff
@@ -57,56 +57,78 @@ void SceneBuilder::RegisterNatives(tonic::DartLibraryNatives* natives) {
 }
 
 SceneBuilder::SceneBuilder()
-    : m_currentLayer(nullptr), m_currentRasterizerTracingThreshold(0) {}
+    : m_currentLayer(nullptr), m_currentRasterizerTracingThreshold(0) {
+  m_cullRects.push(SkRect::MakeLargest());
```

Comment:
We might want to tell the scene building the size of the screen at the start so that its initial cull rect can be tighter.